src/containers/HabitatProject/HabitatProjectContent.tsx

```diff
@@ -1,23 +1,25 @@
 import { Grid } from '@material-ui/core';
-import React from 'react';
+import { SetStateAction, useEffect, useState } from 'react';
+import type { Ibook } from 'src/redux/mapStoreToProps';
 import ELCALogo from '../../components/elcaLogo';
 import PicSlider from '../../components/PicSlider';
 
-const images = [{
-  _id: '5e7273f7349ad8000410f761',
-  title: 'CLC Fellowship',
-  url: 'https://dl.dropboxusercontent.com/s/s1n4nxwc4pz3bml/CLC_Fellowship.png?dl=0',
-  type: 'habitat',
-  comments: '',
-},
-{
-  _id: '5e7273f7349ad8000410f762',
-  title: 'Psalm 127',
-  url: 'https://dl.dropboxusercontent.com/s/w4drwppxa9jrcmo/psalm127.jpg?dl=0',
-  type: 'habitat',
-  comments: '',
-},
-]; // 'showCaption' displays captions if needed'
+// const images = [{
```

Comment:
@kathleenahall please delete the code that you commented out

Reply:
How do I know what the codeclimate issue is?

Reply:
@kathleenahall scroll down and click the details, it has to do with unit test coverage

Reply:
@kathleenahall it looks like details does not show you the test coverage, you need to run the tests locally then open up the /coverage/lcov-report/index.html file in our chrome browser

Reply:
npm version minor is throwing me an error so I'm still working on that, this is what the coverage shows but I'm unsure of what to do next

![Screenshot from 2022-10-31 09-53-33](https://user-images.githubusercontent.com/109632604/199025103-d12aa639-b904-4d18-9955-c214e2107fdf.png)

Reply:
Nvm, I got the version updated

Reply:
@kathleenahall i'm going to approve this merge request, but you will need to next work on learning how to write the unit tests, the above is a unit test coverage report, you can click into the report and it shows you exactly which functions are not having any unit tests. Those will need to be written. Start reading about `react unit testing with jest`